starfish/stack.py

```diff
@@ -823,6 +823,8 @@ def tile_opener(tileset_path, tile, ext):
                     ),
                     "wb")
 
+        if not filepath.endswith('.json'):
```

Comment:
Why is this desirable?  One does not call `imsave("foo",... )` and expect to get foo.tiff